bears/go/GoMetaLintBear.py

```diff
@@ -0,0 +1,66 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='gometalinter',
+        output_format='regex',
+        output_regex=r'(?P<filename>\w+\.go):'
+                     r'(?P<line>\d+):'
+                     r'(?P<column>\d*):'
+                     r'(?P<severity>[a-z]+): '
```

Comment:
Hey, wouldn't it be great if you add severity_map parameter of `@linter` annotation. Refer this http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html#using-severities.

Reply:
No, it won't be great because:
- Unlike whatever is mentioned in the tutorials, there is a clear indication of what the severity level is when gometalinter is invoked. Not just letters.
- There are many severity levels in gometalinter because it's a combination of every Go bear that we currently support and jolting them down to mere three levels vandalizes its purpose.

Still, I'll look into it. ;)

Reply:
Well, I went through the code and I saw that all the messages were mapped to essentially 2 severities, warning and error. https://github.com/alecthomas/gometalinter/blob/master/issue.go. Are there really other severity levels besides these two. Have you tested the linter?

Reply:
Oh, right. I think I confused it with the other linter bear I was working on. :laughing: 
Will add the severity map thing.;)